Sources/AppCoinsSDK/Models/Services/Billing/AppCoinBillingService.swift

```diff
@@ -11,9 +11,9 @@ internal protocol AppCoinBillingService {
     
     func createTransaction(wa: Wallet, raw: CreateAPPCTransactionRaw, completion: @escaping (Result<CreateTransactionResponseRaw, TransactionError>) -> Void)
     
-    func getPaymentMethods(value: String, currency: Coin, result: @escaping (Result<GetPaymentMethodsRaw, BillingError>) -> Void)
+    func getPaymentMethods(value: String, currency: String, result: @escaping (Result<GetPaymentMethodsRaw, BillingError>) -> Void)
     
-    func convertCurrency(money: String, fromCurrency: Coin, toCurrency: Coin?, result: @escaping (Result<ConvertCurrencyRaw, BillingError>) -> Void)
+    func convertCurrency(money: String, fromCurrency: String, toCurrency: String?, result: @escaping (Result<ConvertCurrencyRaw, BillingError>) -> Void)
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done